wagtail/admin/templates/wagtailadmin/reports/listing/_list_page_types_usage.html

```diff
@@ -0,0 +1,93 @@
+                </th>
+                <th class="last-edited-page">
+                    {% trans 'Last edited page' %}
+                </th>
+                <th class="last-edited-at">
+                    {% trans 'Last edit' %}
+                </th>
+                <th class="last-edited-by">
+                    {% trans 'Last edited page owner' %}
+                </th>
+                {% block extra_columns %}
+                {% endblock extra_columns %}
+            </tr>
+        {% endblock post_parent_page_headers %}
+    </thead>
+    <tbody>
+        {% if page_types %}
+            {% for page_type in page_types %}
+                <tr class="{% block page_row_classname %}{% endblock page_row_classname %}">
+                    <td class="page-type" valign="top" data-listing-page-type>
+                        {% block page_type %}
+                            {{ page_type.name|title }}
+                        {% endblock page_type %}
+                    </td>
+                    <td class="app-label" valign="top">
+                        {{ page_type.app_label }}.{{ page_type.model }}
+                    </td>
+                    <td class="count" valign="top">
+                        {% if page_type.count > 0 %}
+                            <a href="{% url 'wagtailadmin_pages:type_use' content_type_app_name=page_type.app_label content_type_model_name=page_type.model %}">{{ page_type.count }}</a>
+                        {% else %}
+                            {{ page_type.count }}
+                        {% endif %}
+                    </td>
+                    <td class="last-edited-page" valign="top">
+                        {% if page_type.last_edited_page %}
+                            <a href="{% url 'wagtailadmin_pages:edit' page_type.last_edited_page.id %}">
+                                {{ page_type.last_edited_page.get_admin_display_title }}
+                            </a>
+                            {% i18n_enabled as show_locale_labels %}
+                            {% if show_locale_labels %}
+                                {% locale_label_from_id page_type.last_edited_page.locale_id as locale_label %}
+                                {% status locale_label classname="w-status--label" %}
+                            {% endif %}
+                        {% else %}
+                            -
+                        {% endif %}
+                    </td>
+                    <td class="last-edited-at" valign="top">
+                        {% if page_type.last_edited_page.latest_revision_created_at %}
+                            {% human_readable_date page_type.last_edited_page.latest_revision_created_at %}
+                        {% else %}
+                            -
+                        {% endif %}
+                    </td>
+                    <td class="last-edited-by" valign="top">
+                        {% if page_type.last_edited_page_owner %}
```

Comment:
It's a single character difference, but saves us a good few lines of Python :)
```suggestion
                        {% if page_type.last_edited_page.owner %}
```

Reply:
Not displaying the owner anymore :)